app/forms/fields/custom_select_field.py

```diff
@@ -0,0 +1,35 @@
+from wtforms import SelectField
+
+
+class CustomSelectField(SelectField):
```

Comment:
We should name custom fields after their functionality rather than just prefixing with `Custom`. This could be  `SelectFieldWithDetailAnswer` and the `MultipleSelectField` could be `MultipleSelectFieldWithDetailAnswer`. I can't think of sensible names for the custom Number and Decimal fields though.